app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/service/DeviceShieldTileService.kt

```diff
@@ -58,7 +58,10 @@ class DeviceShieldTileService : TileService() {
     @Inject lateinit var dispatcherProvider: DispatcherProvider
 
     private var deviceShieldStatePollingJob = ConflatedJob()
-    private val serviceScope = CoroutineScope(dispatcherProvider.io())
+
+    private val serviceScope by lazy {
+        CoroutineScope(dispatcherProvider.io())
+    }
```

Comment:
nit: I think it's fine but I rather use the `Dispatchers.IO` here and suppress the warning with clear comment.

Reply:
Just seen your comment in Asana. Changed to Dispatcher.IO.